function/examples/go.mod

```diff
@@ -2,11 +2,20 @@ module github.com/kyma-incubator/hydroform/function-examples
 
 go 1.14
 
+//replace github.com/kyma-incubator/hydroform/function => github.com/pPrecel/hydroform/function 2bae0404bc9d1ec28031c64776f9f96768b9e878
+replace github.com/kyma-incubator/hydroform/function => github.com/pPrecel/hydroform/function v0.0.0-20210216092427-2bae0404bc9d
```

Comment:
We need to remember to remove this once the [function PR](https://github.com/kyma-incubator/hydroform/pull/186) will be merged and before we merge `examples`.

Reply:
Done